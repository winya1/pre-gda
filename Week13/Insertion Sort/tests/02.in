3
4
4 3 2 1
5
88 2 12 46 90
10
1 2 4 5 6 7 8 9 10 3
