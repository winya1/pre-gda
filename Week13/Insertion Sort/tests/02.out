Case #1:
Current value: 3
[4, 4, 2, 1]
[3, 4, 2, 1]
Current value: 2
[3, 4, 4, 1]
[3, 3, 4, 1]
[2, 3, 4, 1]
Current value: 1
[2, 3, 4, 4]
[2, 3, 3, 4]
[2, 2, 3, 4]
[1, 2, 3, 4]
Sorted: [1, 2, 3, 4]

Case #2:
Current value: 2
[88, 88, 12, 46, 90]
[2, 88, 12, 46, 90]
Current value: 12
[2, 88, 88, 46, 90]
[2, 12, 88, 46, 90]
Current value: 46
[2, 12, 88, 88, 90]
[2, 12, 46, 88, 90]
Current value: 90
[2, 12, 46, 88, 90]
Sorted: [2, 12, 46, 88, 90]

Case #3:
Current value: 2
[1, 2, 4, 5, 6, 7, 8, 9, 10, 3]
Current value: 4
[1, 2, 4, 5, 6, 7, 8, 9, 10, 3]
Current value: 5
[1, 2, 4, 5, 6, 7, 8, 9, 10, 3]
Current value: 6
[1, 2, 4, 5, 6, 7, 8, 9, 10, 3]
Current value: 7
[1, 2, 4, 5, 6, 7, 8, 9, 10, 3]
Current value: 8
[1, 2, 4, 5, 6, 7, 8, 9, 10, 3]
Current value: 9
[1, 2, 4, 5, 6, 7, 8, 9, 10, 3]
Current value: 10
[1, 2, 4, 5, 6, 7, 8, 9, 10, 3]
Current value: 3
[1, 2, 4, 5, 6, 7, 8, 9, 10, 10]
[1, 2, 4, 5, 6, 7, 8, 9, 9, 10]
[1, 2, 4, 5, 6, 7, 8, 8, 9, 10]
[1, 2, 4, 5, 6, 7, 7, 8, 9, 10]
[1, 2, 4, 5, 6, 6, 7, 8, 9, 10]
[1, 2, 4, 5, 5, 6, 7, 8, 9, 10]
[1, 2, 4, 4, 5, 6, 7, 8, 9, 10]
[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
Sorted: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
